automatic/sqlite/update.ps1

```diff
@@ -5,50 +5,50 @@ $releases = 'https://sqlite.org/download.html'
 function global:au_SearchReplace {
    @{
         ".\legal\VERIFICATION.txt" = @{
-          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
-          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
-          "(?i)(\s+Toolsx32:).*"       = "`${1} $($Latest.URLTools32)"
-          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
-          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
-          "(?i)(checksumTools32:).*"   = "`${1} $($Latest.ChecksumTools32)"
+          "(?<=x32:\s).*"             = "$($Latest.URL32)"
+          "(?<=\sx64:\s).*"           = "$($Latest.URL64)"
+          "(?<=\sToolsx64:\s).*"      = "$($Latest.URLTools64)"
+          "(?<=checksum32:).*"        = "$($Latest.Checksum32)"
+          "(?<=checksum64:).*"        = "$($Latest.Checksum64)"
+          "(?<=checksumTools64:\s).*" = "$($Latest.ChecksumTools64)"
         }
     }
 }
 
 function global:au_BeforeUpdate {
     Get-RemoteFiles -Purge -NoSuffix
-    $tools_name = $Latest.URLTools32 -split '/' | Select-Object -Last 1
-    Invoke-WebRequest $Latest.URLTools32 -OutFile tools\$tools_name
-    $Latest.ChecksumTools32 = Get-FileHash tools\$tools_name | ForEach-Object Hash
+    $tools_name = $Latest.URLTools64 -split '/' | Select-Object -Last 1
+    Invoke-WebRequest $Latest.URLTools64 -OutFile tools\$tools_name
+    $Latest.ChecksumTools64 = Get-FileHash tools\$tools_name | ForEach-Object Hash
 }
 
 function global:au_GetLatest {
     function get_version( [int]$Bit=32 ) {
-        $version = $download_page.AllElements | Where-Object tagName -eq 'td' | Where-Object InnerHtml -match "$Bit-bit DLL .+ for SQLite version" | ForEach-Object InnerHtml
-        $version -match '((?:\d+\.)+)' | out-null
-        $Matches[0] -replace '\.$'
+        $re = '(?<=,)(?<Version>.+),(?<Filename>\d*\/sqlite-dll-win-x{0}.+\.zip)' -f ((86, $Bit)[!($Bit -eq 32)])
+        $download_page.Content -match $re | Out-Null
+        $Matches.Version
     }
 
     function url_exists( [string] $Url ) {
         try { ([System.Net.WebRequest]::Create($Url)).GetResponse().Close(); return $true } catch { return $false }
     }
 
-    $download_page = Invoke-WebRequest -Uri $releases
+    $download_page = Invoke-WebRequest -Uri $releases -UseBasicParsing
     $version32 = get_version 32
     $version64 = get_version 64
 
-    $re  = '\-win\d\d\-.+\.zip'
-    $url = $download_page.links | Where-Object href -match $re | ForEach-Object { 'https://sqlite.org/' + $_.href }
-    $url32      = $url -like '*dll*win32*' | Select-Object -First 1
-    $url64      = $url -like '*dll*win64*' | Select-Object -First 1
-    $urlTools32 = $url -like '*tools*win32*'| Select-Object -First 1
+    $re   = '(?<=,)\d{4}/sqlite-.+-win-x\d{2}-\d{7}\.zip'
```

Comment:
Same comment with this regular expression.

Reply:
Reverted to previous code approach sans complex regex.